docs/pages/server-access/guides/openssh.mdx

```diff
@@ -69,6 +69,100 @@ Teleport OpenSSH supports:
 - IPv4 `ubuntu@184.45.45.30`
 - IPv6 `root@2001:db8::2`
 
+## Using OpenSSH client
```

Comment:
This section is moved from the User Manual but is mostly redundant here. I will prune this quite a bit.